src/frontend/src/lib/utils/auth.utils.ts

```diff
@@ -39,3 +39,11 @@ export const isAlternativeOrigin = (): boolean => {
 };
 
 export const hasDerivationOrigin = (): boolean => notEmptyString(AUTH_DERIVATION_ORIGIN);
```

Comment:
This does not require to be exposed anymore?

Reply:
same for `isAlternativeOrigin`?

Reply:
Yes, good catch. Done!